db/seeds.rb

```diff
@@ -3,19 +3,18 @@
 # This file should contain all the record creation needed to seed the database with its default values.
 # The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
 
-unless Rails.env.production?
+# Prevent executing this code in the production Rails env, which is what we use in deployed environments.
+if Rails.env.local?
   require_relative '../spec/support/create_strategy_repository_pattern'
   require_relative '../spec/support/item_method_sender'
   require_relative '../spec/support/apo_method_sender'
   require_relative '../spec/support/reset_solr'
 
-  $stdout.puts 'This will clear the Solr repo. Are you sure? [y/n]:'
-  if gets.chomp == 'y'
```

Comment:
Forcing the `db:seed` task to be used in an interactive mode breaks tasks that depend on it, such as `db:prepare`. I could be convinced that automatically clearing Solr in dev/test is not the best idea, in which case I'd probably take a different approach like sniffing for an opt-in env variable that communicates a desire to clear Solr. Open to other ideas too.